### DISCUSSION TITLE
Posts are cut off after "read more"?

### DISCUSSION THREAD

**Discussion by DwellOnDreams93** [TumblThreeApp/TumblThree#468] [Q&A (Question & Answer)]
Trying to backup my tumblr likes and everything, and any post that has a "read more" option, TumblThree cuts off at that point. Is there any way to work around that?

**Comment by thomas694** [comment]
Is that "read more" (literally?) a thing which you see on all longer posts or only on posts from some blogs? Can you provide an URL of an example post? If it's safe then here, else in the feedback dialog/page.

**Reply by DwellOnDreams93** [reply]
It's generally on longer posts; the person creating the post has the option of adding it (so that the rest of the post is under a cut that reads "expand" or "read more"). This is the url for one but since you're clicking the link to the post, it may just show the entire post. 

https://www.tumblr.com/the-modern-typewriter/727100101466767361/hi-can-you-do-a-villain-x-villain?source=share


And another, just because I'm not entirely sure if "read more" and expand are different: 
https://www.tumblr.com/im-totally-not-an-alien-2/702715441745952768/do-you-require-assistance-phantom-perked-up-at?source=share

**Reply by thomas694** [reply]
I added them both to my likes and downloaded them. It didn't matter if I downloaded metadata as json or text, in both cases the text were complete. On the likes page the text post is shown with "Keep reading", the answer post with "Expand".

How do you open the files to read the texts? I opened them in a JSON viewer, but obviously that app isn't capable of displaying the whole text of the fields. One text was cut off at an internal newline, the other in the middle of a sentence. In my text editor the texts were complete.
Could it be that you have a similar issue?
Can you please look into the files (texts.txt, answers.txt...) with an appropriate text editor and check whether the texts are really cut off or not?
